### DISCUSSION TITLE
Hiding environmental variable

### DISCUSSION THREAD

**Discussion by zeymx** [parcel-bundler/parcel#6914] [Q&A]
I'm trying to build a project with Parcel and hide environmental variables from an output javascript file. How can I do it?

**Comment by zeymx** [comment]
I found out that I need to use:

```
"engines": {
        "node": ">=12" 
      },
```

Although I added this line of code: `"includeNodeModules": true`
But I keep getting this error message:

```
@parcel/packager-js: External modules are not supported when building for browser

  ..\node_modules\axios\lib\adapters\http.js:7:20  
    6 | var buildURL = require('./../helpers/buildURL');
  > 7 | var http = require('http');
  >   |                    ^^^^^^
    8 | var https = require('https');
    9 | var httpFollow = require('follow-redirects').http;
```

**Comment by LiberQuack** [comment]
I also use axios, but it works fine here

I am using parcel version "2.0.0-nightly.837"
